### ISSUE TITLE
[Case Studies] Qonto

### ISSUE THREAD

**Issue by locks** [mainmatter/mainmatter.com#1229]
We are adding a new case study to the website about Qonto.
It should match the current design of the other case studies, and not be blocked by the redesign effort.
This is so that we can quickly ship the case study and re-use as much of the current code as possible, with a follow-up task taking care of the redesign.

## Tasks
- Create qonto case study page component
- Add route map entry for page
- Fill in final copy
- Implement quote section
- Add optimized images for 600px and 1200px breakpoint

##  Resources
- [Abstract](https://share.goabstract.com/1592c316-fa9d-412d-8cf9-0849b379e0f9)

**Comment by marcoow** [comment]
done via #1228
